src/Operator/Constructors.jl

```diff
@@ -169,17 +278,26 @@ function GalleryOperator(
     mesh::Mesh;
     collocatedquadrature::Bool = false,
     mapping::Union{Tuple{Function,Function},Nothing} = nothing,
+    parameters::Union{NamedTuple,Nothing} = nothing,
 )
     if haskey(operatorgallery, name)
+        numbercomponents = 1
```

Comment:
Why? 

This doesn't make sense. The number of components here must always be 1, so this change doesn't help us. I would revert this commit.

Reply:
I am only trying to implement what I think I understand from our conversations. I will reset back to previous commit